### DISCUSSION TITLE
How to use proxy header

### DISCUSSION THREAD

**Discussion by painterner** [filebrowser/filebrowser#1483] [General]
I tried to set:

> filebrowser config set --auth.method=proxy --auth.header=admin:admin

However, how to login? I can't login with admin/admin anymore.

**Comment by ramiresviana** [comment]
The Proxy Header authentication method is used with a proxy server. Check the documentation at https://filebrowser.org/configuration/authentication-method#proxy-header

**Reply by painterner** [reply]
Tks
